typed-racket-lib/typed-racket/typecheck/tc-expr-unit.rkt

```diff
@@ -151,6 +151,9 @@
       [stx:exn-handlers^
        (register-ignored! form)
        (check-subforms/with-handlers form expected) ]
+      [(~and stx:typed-racket:ignore-type-information^ (#%expression inner))
+       (tc-expr/check/internal #'inner Univ)
+       (ret Univ -true-propset -empty-obj)]
```

Comment:
This line is wrong, it should be `(ret Univ (-PS -tt -tt) -empty-obj)`.

cc @mflatt